core/src/main/scala/cats/Select.scala

```diff
@@ -0,0 +1,13 @@
+package cats
+
+import simulacrum.{typeclass}
+import scala.annotation.implicitNotFound
+
+@implicitNotFound("Could not find an instance of Select for ${F}")
+@typeclass trait Select[F[_]] extends Apply[F] {
+  def select[A, B](fab: F[Either[A, B]])(ff: F[A => B]): F[B] =
+    map2(fab, ff) {
```

Comment:
I don't like this default since it is no more powerful than Apply.

Since FlatMap is so rarely used as a typeclass I would suggest removing Select and just make Monad extend Selective.